README.md

```diff
@@ -1,6 +1,6 @@
 <img width="600" src="https://rawgit.com/Leaflet/Leaflet/master/src/images/logo.svg" alt="Leaflet" />
 
-Leaflet is the leading open-source JavaScript library for **mobile-friendly interactive maps**.
+Leaflet is the leading open-source ([BSD 2-Clause license][]) JavaScript library for **mobile-friendly interactive maps**.

```

Comment:
```suggestion
Leaflet is the leading open-source ([BSD 2-Clause license]) JavaScript library for **mobile-friendly interactive maps**.
```

(not sure that it is best place to link license, but you do not need those empty brackets anyway.

Reply:
I am aware about the brackets, i was just trying to stick to the way the rest of the links were also written in the md.